devicetypes/aamann/enlighten-envoy-local.src/enlighten-envoy-local.groovy

```diff
@@ -296,9 +302,10 @@ def ping() {
 
 def startPoll() {
 	unschedule()
-	// Schedule 2 minute polling
+	def wantMin = 6       // Schedule N minute polling Integer < 60
```

Comment:
> So it seems you need to ensure on every time running, some variable (even a last timer run) needs to be updated.

I do that (actual energy data if new data is available, a dummy `lastUpdate` value id data is stale). However, that happens in the `parse()` method as getting a response there is the only way to know that the device is still alive. The problem is that the `parse()` function doesn't get called reliably (it would list either `[…] new data: […]` or `[…] no new data`).